config/3.3/event-manager/templates/catalogsource.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Another reason that I'm thinking maybe we should combine `ai-manager` and `event-manager` to a single chart. We have duplication such as `CatalogSource`, `check-xyz-prereqs` and `configure-aiops-network-policy` job, etc. in both charts.

Reply:
Our template should also be adapted to the installation of internal build. Internal build cataloug images of aimanager and eventmanager are not same, which will not be able to use a catalogsource file

Reply:
For internal build, can we make it conditionally when writing helm template.

Reply:
Of course, but under the premise that AIManager and eventmanager are installed separately, these duplications are unavoidable